### ISSUE TITLE
Change project card hover effect

### ISSUE THREAD

**Issue by prathameshbhalekar** [anitab-org/anitab-org.github.io#225]
### Change project card hover effect
Currently we are transforming scale of card on hover which looks very dull. It can be changed to something like darkening the background to make it look more modern.

### To Reproduce
1. Go to 'https://anitab-org.github.io/'
2. Click on 'projects'
3. Scroll down to 'projects cards'

### Expected behavior
It could be replaced with a more modern effect like darken on hover.

![ezgif-4-a3809b3cb6d2](https://user-images.githubusercontent.com/33856767/107107584-f7934780-6857-11eb-81b2-b14164eca901.gif)

### Screenshots
![ezgif-4-c73c98392753](https://user-images.githubusercontent.com/33856767/107107453-0b8a7980-6857-11eb-9b4d-7ffb948aaef9.gif)

### Desktop:
 - OS: Windows 10
 - Browser Firefox
 - Version 85.0.1 (64-bit)

### Additional context
Can I work on this issue if approved?

**Comment by Srashtimittal** [comment]
@prathameshbhalekar Can I work on this issue?

**Comment by prathameshbhalekar** [comment]
> @prathameshbhalekar Can I work on this issue?

Even I'm new here but I think it needs to be approved first.

**Comment by Amulya-coder** [comment]
@prathameshbhalekar Thanks for showing interest in the project. I think it's a valid feature but it's for future work currently our main focus is on the web site. Dark mode functionality is going to be implemented afterward, but you can also confirm with the mentor Thanks 😊.

**Comment by prathameshbhalekar** [comment]
> @prathameshbhalekar Thanks for showing interest in the project. I think it's a valid feature but it's for future work currently our main focus is on the web site. Dark mode functionality is going to be implemented afterward, but you can also confirm with the mentor Thanks 😊.

I didn't mean dark mode. I was referring to the hover effect of project card. Can you please tell me which part made you think I was talking about dark mode so that I can change it? Thanks in advance 👍

**Comment by Rahulm2310** [comment]
@prathameshbhalekar Although the project card scaling on hover is not smooth, still it is not an issue. For now it can be put on hold. Thanks 🙂

**Comment by prathameshbhalekar** [comment]
> @prathameshbhalekar Although the project card scaling on hover is not smooth, still it is not an issue. For now it can be put on hold. Thanks 🙂

Should I close the issue?

**Comment by annabauza** [comment]
this is up for design team to discuss. This kind of issues should've been risen on design session first.
